src/node/blockstorage.cpp

```diff
@@ -11,6 +11,7 @@
 #include <flatfile.h>
 #include <fs.h>
 #include <hash.h>
+#include <node/mempool_persist_args.h>
```

Comment:
c4249c1243981f42091950d1295c2be6bd4bf118: Why is this needed?

Reply:
Fixed in 06b88ffb8a